app/views/step_by_step_pages/index/_results.erb

```diff
@@ -1,41 +1,42 @@
-    <tr>
-      <th class="govuk-table__header govuk-!-font-weight-bold" scope="col">Title</th>
-      <th class="govuk-table__header govuk-!-font-weight-bold" scope="col">Status</th>
-      <th class="govuk-table__header govuk-!-font-weight-bold" scope="col">Base path / preview</th>
-    </tr>
-  </thead>
+<%= GovukPublishingComponents::AppHelpers::TableHelper.helper(self, "Documents list", { sortable: true, caption_classes: "govuk-visually-hidden" }) do |t| %>
 
-  <tbody class="govuk-table__body">
-    <% if @step_by_step_pages.present? %>
-      <% @step_by_step_pages.each do |step_by_step_page| %>
-        <tr class="govuk-table__row" data-status="<%= step_by_step_page.status[:name] %>">
-          <td class="govuk-table__cell">
-            <strong><%= link_to step_by_step_page.title, step_by_step_page %></strong>
-          </td>
-          <td class="govuk-table__cell">
-            <span class="h4">
-              <span class="label <%= step_by_step_page.status[:label_class] %>" title="<%= step_by_step_page.status[:text] %>"><%= step_by_step_page.status[:text] %></span>
-            </span>
-          </td>
-          <td class="govuk-table__cell">
-            <% if step_by_step_page.has_draft? %>
-              <%= link_to "/#{step_by_step_page.slug}", preview_url(step_by_step_page.slug, auth_bypass_id: step_by_step_page.auth_bypass_id), target: "_blank" %>
-            <% elsif step_by_step_page.status[:name] == "live" %>
-              <%= link_to "/#{step_by_step_page.slug}", live_url(step_by_step_page.slug), target: "_blank" %>
-            <% else %>
-              /<%= step_by_step_page.slug %>
-            <% end %>
-          </td>
-        </tr>
+  <%= t.head do %>
+    <%= t.header "Title" %>
+    <%= t.header "Status" %>
+    <%= t.header "Last updated" %>
+  <% end %>
+
+  <%= t.body do %>
+    <% @step_by_step_pages.each do |step_by_step_page| %>
+
+      <% preview_link = capture do %>
+        <% if step_by_step_page.has_draft? %>
+          <%= link_to "/#{step_by_step_page.slug}", preview_url(step_by_step_page.slug, auth_bypass_id: step_by_step_page.auth_bypass_id), target: "_blank", class: "govuk-link" %>
+        <% elsif step_by_step_page.status.published? %>
+          <%= link_to "/#{step_by_step_page.slug}", published_url(step_by_step_page.slug), target: "_blank", class: "govuk-link" %>
+        <% else %>
+          /<%= step_by_step_page.slug %>
+        <% end %>
+      <% end %>
+
+      <%= t.row do %>
+        <%= t.cell render "govuk_publishing_components/components/document_list", {
+          items: [
+            {
+              link: {
+                text: step_by_step_page.title,
+                path: step_by_step_page
+              },
+              metadata: {
+                preview_link: preview_link
+              }
+            }
+          ]
+        } %>
+
+        <%= t.cell step_by_step_page.status.humanize %>
+        <%= t.cell step_by_step_page.created_at.strftime('%-d %B %Y') %>
```

Comment:
Would probably be best to create a new helper method in TimeOptionsHelper and call that from here. There's a [`format_full_date_and_time`](https://github.com/alphagov/collections-publisher/blob/f8401b53dd3f48766d63dc4b9f050a856d190fa7/app/helpers/time_options_helper.rb#L19-L21) you can use for inspiration, probably call this one `format_full_date` or something?

Reply:
Updated to use the time_options helper